index.html

```diff
@@ -148,48 +165,70 @@ <h2 class="proj-title">Multi-Post Stories</h2>
-          <li class="about-item d-flex border font500">Frameworks<img src="icon/chevron-right.png" alt="chevron right"/></li>
-          <li class="about-item d-flex border font500">Skills<img src="icon/chevron-right.png" alt="chevron right"/></li>
+          <li class="about-item d-flex font500" tabindex="0">Languages
+            <img src="icon/chevron-down.png" alt="chevron down"/></li>
+          <li class="lang-item d-flex font500" tabindex="0">
+            <img src="icon/javascript.png" class="pad10" alt="JavaScript"/>
+            <span class="pad10">JavaScript</span></li>
+          <li class="lang-item d-flex font500" tabindex="0">
+            <img src="icon/html.png" class="pad10" alt="HTML"/>
+            <span class="pad10">HTML</span></li>
+          <li class="lang-item d-flex font500" tabindex="0">
+            <img src="icon/css.png" class="pad10" alt="CSS"/>
+            <span class="pad10">CSS</span></li>
+          <li class="about-item d-flex border font500" tabindex="0">Frameworks<img
+              src="icon/chevron-right.png" alt="chevron right" /></li>
+          <li class="about-item d-flex border font500" tabindex="0">Skills<img
+              src="icon/chevron-right.png" alt="chevron right" /></li>
         </ul>
       </div>
     </section>
     <footer class="contact d-flex">
-      <h2 class="white bold font40-52">Contact me</h2>
-      <p class="white font20-28">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
-      <form action="https://formspree.io/f/xdoypzpo" method="POST" class="form d-flex">
-        <input class="usr_detail smooth_border" type="text" name="usr-name" required maxlength="30" placeholder="Your name">
-        <input class="usr_detail smooth_border" type="email" name="usr-email" required placeholder="name@mail.com">
-        <textarea class="usr_comment smooth_border" name="usr-message" required maxlength="500" placeholder="Write your message here"></textarea>
+      <h2 class="white bold font40-52" tabindex="0">Contact me</h2>
+      <p class="white font20-28 bold" tabindex="0">If you have an application you are
+        interested in developing, a feature that you need built or a project
+        that needs coding. I’d love to help with it</p>
+      <form action="https://formspree.io/f/xdoypzpo" method="POST" class="form
+        d-flex">
+        <input class="usr_detail smooth_border" type="text" name="usr-name"
+          required maxlength="30" placeholder="Your name">
+        <input class="usr_detail smooth_border" type="email" name="usr-email"
+          required placeholder="name@mail.com">
+        <textarea class="usr_comment smooth_border" name="usr-message" required
+          maxlength="500" placeholder="Write your message here"></textarea>
         <button class="button 500" type="submit">Get It Touch</button>
       </form>
```

Comment:
- [ ] Currently, the button is failing the minimum requirements for contrast ratio. Kindly refactor the design to pass the test. You can use this [tool](https://www.aditus.io/button-contrast-checker/) to check.

Reply:
We have doubts about this because we need to stick with the Figma layout.